build/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Check PHP Version
+required_php_version="8.1"
+
+installed_php_version=$(php -r "echo PHP_MAJOR_VERSION.'.'.PHP_MINOR_VERSION;")
+if [ "$installed_php_version" != "$required_php_version" ]; then
+    echo "Error: PHP version $required_php_version is required, but you have PHP $installed_php_version installed."
+    exit 1
+fi
+
+# Download PHP Client Code
+github_repo="https://github.com/aerospike/php-client"
+branch="CLIENT-2422-Code-completion"
```

Comment:
I believe This should be updated

Reply:
have updated it to take a pull from main. have tested the make file against php-rs branch, and will test against main branch as soon its merged